components/contribution-flow/index.js

```diff
@@ -663,6 +663,7 @@ class ContributionFlow extends React.Component {
                   )}
 
                   <ContributionFlowStepContainer
+                    LoggedInUser={LoggedInUser}
```

Comment:
Same here, passing down props like `LoggedInUser` is tedious. Helpers like `useUser` and `withLoggedInUser` are here to make things simpler

Reply:
Ops, somehow I forgot the use of hooks. 🤯